### ISSUE TITLE
Simple documentation for curators

### ISSUE THREAD

**Issue by cappadona** [cul-it/exhibits-library-cornell-edu#25]
> _Imported from Jira_
> https://issues.library.cornell.edu/browse/SPOT-19

**Comment by starsplatter** [comment]
https://groups.google.com/forum/#%21topic/blacklight-development/ZfS6CRHN0_I

**Comment by mhk33** [comment]
https://github.com/blog/2233-publish-your-project-documentation-with-github-pages
https://www.youtube.com/watch?v=2MsN8gpT6jY

**Comment by mhk33** [comment]
Branch from master and do pull requests back into master, avoid dev and unfinished work.

**Comment by mhk33** [comment]
I started the documentation branch, which is branched off master:
https://github.com/cul-it/exhibits-library-cornell-edu/tree/documentation

Here's the barebones beginning of documentation:
https://cul-it.github.io/exhibits-library-cornell-edu/

**Comment by mhk33** [comment]
@cappadona @starsplatter I finished both basic documentation and addressing the various issues that were included in this epic (I think). Please let me know if you have any other suggestions:
https://cul-it.github.io/exhibits-library-cornell-edu/

**Comment by cappadona** [comment]
@mhk33 This looks great. The only suggestion I have, and I think this is something we've already discussed, is to add an outline/nav of the sections.

I did a quick search and it looks like the [Leap Day theme](https://pages-themes.github.io/leap-day) supports auto-generation of an outline of `h1` and `h2` headings. We could either switch over to that to test it out or override the default layout of the minimal theme and [borrow the JS](https://github.com/pages-themes/leap-day/blob/4d2d418de73cc3f3cf5fd4656207cf73ff466e35/assets/js/main.js#L16-L28) from Leap Day if we want to get fancy.
